sys-cluster/hpx/hpx-1.11.0.ebuild

```diff
@@ -33,7 +33,7 @@ BDEPEND="
 "
 RDEPEND="
 	${PYTHON_DEPS}
-	>=dev-cpp/asio-1.12.0
+	<=dev-cpp/asio-1.33
```

Comment:
<= like this is suspicious as it won't cover 1.33-r1, but also, why do we need this? Version restrictions (of the < variety) should always come with a bug reference and some comment above it linking to that bug.

Reply:
It doesn't build against >= asio-1.34. There's an issue for that: https://github.com/STEllAR-GROUP/hpx/issues/6730. It will be fixed in hpx-1.11.1.

Reply:
Please mention that in the commit message & in a comment above the dep bloc. You should also do <asio-1.34 instead.